app/helpers/textual_mixins/textual_devices.rb

```diff
@@ -1,129 +1,130 @@
-require 'textual_mixins/device'
```

Comment:
removing this makes me nervous.
(but that is not the first time I've said something like this and will probably not be the last)
I guess this is what zeitwerk buys us.

Reply:
Is this what zeitwerks calls "opening an existing namespace"?

Reply:
What makes you nervous?   Removing the require or removing the collapsed namespace?

The require was required before because we couldn't autoload it before because it was incorrectly named.  We can so with the change above in app/helpers/textual_mixins/device.rb so the require is not needed.

Re: collapsed namespace... I can double check if that change is needed.  I don't mind keeping it if it works.

Reply:
no, it was the remove of the require.
sounds good